src/magic/actmagic.cpp

```diff
@@ -1909,6 +1909,8 @@ void actMagicMissile(Entity* my)   //TODO: Verify this function.
 					{
 						if (hit.entity->behavior == &actDoor)
 						{
+                            playSoundEntity(hit.entity, 91, 64);
+                            hit.entity->skill[5] = 0; // Unlock the door.
```

Comment:
Out of curiosity, is there any `Sint32& ...` defined for this in Entity.hpp? Like `Sint32& doorLocked`?
The chest has a bunch of these sorts of defines, like `Sint32& chestOpener` IIRC

Reply:
As far as I can see, there is not. It is however possible that `chest_opener(skill[5])` could be used, however for now, I'll keep it unmodified.